tests/SchemaTest.php

```diff
@@ -39,10 +30,10 @@ public function valid_documents_pass(DOMDocument $document, string $schema) : vo
      * @test
      * @dataProvider invalidFileProvider
      */
-    public function invalid_documents_fail(DOMDocument $document, string $schema, array $expected) : void
+    public function invalid_documents_fail(DOM $document, string $schema, array $expected) : void
     {
         $result = $document->relaxNGValidate($schema);
```

Comment:
may want to pass in explcitly `$schematron = true` as this is the place where people looking for where Schematron gets used will look for:
 https://github.com/Erebot/DOM/blob/c97ff2d3e33168fa37626263682400effc62ba76/src/DOM.php#L83